identity/client/e2e/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Not related to this PR but I think it's a good idea to spend some time to migrate to ESLint v9 flatconfig